### PR TITLE
Add next mature card num in dashboard

### PR DESCRIPTION
**Pull request by ryonryon**

### CODE REVIEW

**Comment by ryonryon** [comment]
@axross Can you check this change?

**Comment by axross** [comment]
Set assignee, project and milestone.

File: lib/src/entities/card.dart [hunk 1]
```diff
@@ -13,4 +13,6 @@ abstract class Card with Identifiable<Card, String> {
   double get maturity => Review.calculateMaturity(nextReviewInterval);
 
   bool get isScheduled => nextReviewScheduledAt.isBefore(DateTime.now());
+
+  double get modifier;
```
Comment (axross) [inline lib/src/entities/card.dart:17]: :+1:

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 2]
```diff
@@ -25,6 +25,14 @@ class Summary extends StatelessWidget {
       totalCardsNumber *
       100;
 
+  List getNextMatureCards() => cardsMaturity
+      .where((iter) =>
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Set proper name for elements in the loop

**Review by axross — APPROVED** [review]
More or less perfect!

**Pull request merged**